customTests/azure_common.nhc

```diff
@@ -33,6 +33,40 @@ function pass(){
    fi
 }
 
+function warning(){
+   IFS=$' \t\n'
+   local RET="$1"
+   shift
+
+   CHECK_SUCCESS=0
+
+   if [[ -z "$CHECK" ]] && CHECK="$FUNCNAME $*"; then
```

Comment:
Do you want $FUNCNAME to return warning here? If you want the name of the function calling warning, you need ${FUNCNAME[1]}.

Reply:
Oh good catch let me take a look

Reply:
addressed